### PR TITLE
Init and add api doc

### PR DESCRIPTION
**Pull request by shaojunda**

### CODE REVIEW

**Comment by duanyytop** [comment]
Please use git commit format: https://www.conventionalcommits.org/en/v1.0.0-beta.2/

File: server/config/database.yml.example [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+#
+# You can use this database configuration with:
+#
+#   production:
+#     url: <%= ENV['DATABASE_URL'] %>
+#
+production:
+  <<: *default
+  database: ckb_explorer_production
+  username: <%= ENV.fetch("DB_USERNAME") { "postgres" } %>
```
Comment (classicalliu) [inline server/config/database.yml.example]: `username` and `password` may also required in `development` and `test` env.

Comment (shaojunda) [reply server/config/database.yml.example]: can be empty in development and test env.

Comment (classicalliu) [reply server/config/database.yml.example]: Does database `username` should equal to current system username if left empty?

**Pull request merged**